### PR TITLE
Update Intellij-openshift-connector

### PR DESCRIPTION
**Pull request by sbouchet**

### CODE REVIEW

**Comment by sbouchet** [comment]
/pj-rehearse pull-ci-redhat-developer-intellij-openshift-connector-main-e2e

**Comment by sbouchet** [comment]
/pj-rehearse pull-ci-redhat-developer-intellij-openshift-connector-main-e2e

**Comment by sbouchet** [comment]
/pj-rehearse pull-ci-redhat-developer-intellij-openshift-connector-main-e2e

**Comment by sbouchet** [comment]
/pj-rehearse pull-ci-redhat-developer-intellij-openshift-connector-main-e2e

**Comment by sbouchet** [comment]
/pj-rehearse pull-ci-redhat-developer-intellij-openshift-connector-main-e2e

File: ci-operator/config/redhat-developer/intellij-openshift-connector/redhat-developer-intellij-openshift-connector-main.yaml [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
-    tag: "1.12"
+  project_image:
+    dockerfile_literal: |
+      FROM registry.access.redhat.com/ubi8/ubi
+      RUN yum install -y java-11-openjdk-devel git
 releases:
+  initial:
+    candidate:
+      product: ocp
+      stream: nightly
```
Comment (jeffmaury) [inline ci-operator/config/redhat-developer/intellij-openshift-connector/redhat-developer-intellij-openshift-connector-main.yaml:10]: That will make your PR check very unstable ?

File: ci-operator/config/redhat-developer/intellij-openshift-connector/redhat-developer-intellij-openshift-connector-main.yaml [hunk 2]
```diff
@@ -1,14 +1,19 @@
 build_root:
-  image_stream_tag:
-    name: ubi8-openjdk-11
-    namespace: openshift
-    tag: "1.12"
+  project_image:
+    dockerfile_literal: |
+      FROM registry.access.redhat.com/ubi8/ubi
```
Comment (jeffmaury) [inline ci-operator/config/redhat-developer/intellij-openshift-connector/redhat-developer-intellij-openshift-connector-main.yaml:4]: Same remark, we should use a specific version

**Comment by sbouchet** [comment]
@jeffmaury : well, i'm just trying to get something working and then stabilize :D

**Comment by sbouchet** [comment]
/pj-rehearse pull-ci-redhat-developer-intellij-openshift-connector-main-e2e

**Comment by sbouchet** [comment]
/pj-rehearse pull-ci-redhat-developer-intellij-openshift-connector-main-e2e-openshift

**Comment by sbouchet** [comment]
/pj-rehearse pull-ci-redhat-developer-intellij-openshift-connector-main-e2e-openshift

**Comment by sbouchet** [comment]
/pj-rehearse pull-ci-redhat-developer-intellij-openshift-connector-main-e2e-openshift

**Comment by sbouchet** [comment]
/pj-rehearse pull-ci-redhat-developer-intellij-openshift-connector-main-e2e-openshift

**Comment by sbouchet** [comment]
/pj-rehearse pull-ci-redhat-developer-intellij-openshift-connector-main-e2e-openshift

**Pull request closed without merging**